device/ospfv3/interface/interface.yaml

```diff
@@ -0,0 +1,201 @@
+        area:
+          description: |-
+            The Area ID of the area to which the attached network belongs.
+            All routing protocol packets originating from the interface are
+            labelled with this Area ID.           
+          $ref: '#/components/schemas/Ospfv3Interface.Area'
+          x-field-uid: 3
+        network_type:
+          description: |- 
+            The OSPF network link type.
+          $ref: '#/components/schemas/Ospfv3Interface.NetworkType'
+          x-field-uid: 4
+        capabilities:
+          description: >-
+            Container for OSPFv3 interface capabilities.
+          $ref: '#/components/schemas/Ospfv3Interface.Capabilities'
+          x-field-uid: 5
+        options:
+          description: >-
+            Container for OSPFv3 optional interface properties.
+          $ref: '#/components/schemas/Ospfv3Interface.Options'
+          x-field-uid: 6
+        instance_id:
+          description: |-
+            Enables multiple instances of OSPF to be run over a single link.
+            Each protocol instance should be assigned a separate Instance ID;
+            the Instance ID has link-local significance only.
+          type: integer
+          format: uint32
+          default: 0
+          maximum: 255
+          x-field-uid: 7
+
+    
+    Ospfv3Interface.Area:
+      description: |-
+        Container for OSPFv3 Area ID identifies the routing area to which the host belongs.      
+      type: object
+      properties:
+        choice:
+          description: |-
+            The OSPFv3 Area ID identifies the routing area to which the host belongs. Area ID type can be following format.
+            - id: A 32-bit number identifying the area.
+            - ip: The Area ID in IPv4 address format.
+          type: string
+          default: id
+          x-field-uid: 1
+          x-enum:
+            id:
+              x-field-uid: 1
+            ip:
+              x-field-uid: 2
+        id:
+          description: >-
+            The Area ID.
+          type: integer
+          format: uint32
+          default: 0
+          x-field-uid: 2
+        ip:
+          description: >-
+            The Area ID in IPv4 address format.
+          type: string
+          format: ipv4
+          x-field-uid: 3
+
+    Ospfv3Interface.NetworkType:
+      description: >-
+        The OSPFv3 network link type options.
+          - Broadcast
+          - Point to Point
+      type: object
+      properties:
+        choice:
+          type: string
+          default: broadcast
+          x-field-uid: 1
+          x-enum:
+            broadcast:
+              x-field-uid: 1
+            point_to_point:
+              x-field-uid: 2
+        broadcast:
+          description: >-
+            Capabilities associated with network type broadcast.
+          $ref: '#/components/schemas/Ospfv3Interface.Broadcast'
+          x-field-uid: 2
+        
+    Ospfv3Interface.Capabilities:
```

Comment:
This may not be right to say capabilities ---> may be call it "hello_timers" and then "link_metric" and "validate_received_mtu" flat members just like "instance_id

Reply:
changed `Ospfv3Interface.Capabilities` to `Ospfv3Interface.Advanced` and also the associated ref.